<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1619</width>
    <height>952</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="GraphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>711</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerName1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerName2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionLabel2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ThrowDice">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Throw DIce</string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionLabel4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>860</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position : </string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionLabel3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>860</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position : </string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionLabel1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position : </string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerName4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>830</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerName3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>830</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionStore">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionStore_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionStore_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>860</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="PositionStore_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>860</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QComboBox" name="NumPlayers">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Player ü•≤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Players üïπÔ∏è</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Players ü™ú</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Players üêç</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="NumPlayersLabel">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Players</string>
    </property>
   </widget>
   <widget class="QLabel" name="BoardSizeLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size of Board</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BoardSize">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DiffLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Difficulty</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Diff">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Easy ü™úü™úüêç</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium üêçüêçü™úü™ú</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard üêçüêçüêçüêçü™ú</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="MessageLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RandomBoard">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>360</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Board ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="ThrowDiceMessage">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>280</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1619</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
